include/seqan3/argument_parser/validators.hpp

```diff
@@ -165,10 +166,15 @@ class arithmetic_range_validator
  *
  * \details
  *
- * On construction, the validator must receive a list (vector) of valid values.
+ * On construction, the validator must receive a range or parameter pack of valid values.
  * The struct than acts as a functor, that throws a seqan3::parser_invalid_argument
  * exception whenever a given value is not in the given list.
  *
+ * \note In order to simplify the chaining of validators, all ranges of arithmetic types resolve to the validator
+ *       value type `double` and all ranges of types which are convertible to resolve to the value type std::string.
```

Comment:
```suggestion
* \note In order to simplify the chaining of validators, the option value type is deduced to `double` for ranges whose value type models seqan3::arithmetic, and to `std::string` if the ranges value type is convertible to it.
Otherwise, the option value type is deduced to the value type of the range.
```